src/components/Radio/RadioGroup.tsx

```diff
@@ -32,6 +32,15 @@ export const RadioGroup = <T extends { label?: string; value?: string }>({
 
   const activeItem = items.find((_, i) => i === active);
 
+  const initial = items.findIndex(el => {
+    const value = itemValuePath ? el[itemValuePath] : el.value;
```

Comment:
What do you think about cleaning this up similar to the `Select` component by removing itemValuePath & itemLabelPath?

Reply:
done